server/src/main/java/org/elasticsearch/search/aggregations/bucket/geogrid/GeoGridAggregationBuilder.java

```diff
@@ -154,7 +151,7 @@ public int shardSize() {
 
         if (requiredSize <= 0 || shardSize <= 0) {
             throw new ElasticsearchException(
-                    "parameters [required_size] and [shard_size] must be >0 in geohash_grid aggregation [" + name + "].");
+                    "parameters [required_size] and [shard_size] must be > 0 in geo-grid aggregation [" + name + "].");
```

Comment:
A generic `geo-grid` can probably be replaced with `" + getType() + "`

Reply:
good call!